content/en/blog/2019/announcing-1.3/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Announcing Istio 1.3
+subtitle: Major Update
+description: Istio 1.3 release announcement.
+publishdate: 2019-09-10
+attribution: The Istio Team
+release: 1.3.0
+---
+
+We are pleased to announce the release of Istio 1.3!  
+
+{{< relnote linktonote="true" >}}
+
+The theme of Istio 1.3 is User Experience:
+
+- Improve the experience of new users adopting Istio
+- Improve the experience of users debugging problems
+- Support more applications without any additional configuration
+
+Every few releases, the Istio team gets excited as we make dramatic improvements to usability, APIs, and the overall system performance. This release is one of those where the team is excited to roll out some key updates:
+
+*Intelligent protocol detection*: In past releases, you had to explicitly declare the protocol for service ports by using a special port naming format. This requirement caused problems for users that did not name their ports when they added their applications to the mesh. Starting with 1.3, the protocol on the wire will be automatically detected (as HTTP or TCP), eliminating the need to name ports explicitly. We will be adding support for additional [non-HTTP] protocols in future releases. 
+
+*Mixer-less telemetry*: Yes you read that right! We had already turned off the `istio-policy` service by default as we implemented most of the common security policies such as RBAC directly in Envoy. We are now on track to migrate most of the mixer's telemetry functionality as well into Envoy. As of this release, if all you care about is telemetry for HTTP services, we have enhanced the Istio proxy to emit HTTP metrics directly to Prometheus, without requiring the `istio-telemetry` service to enrich the information. You can experiment with the mixer-less telemetry feature for HTTP services by following these [instructions](https://github.com/istio/istio/wiki/Mixerless-HTTP-Telemetry). We will be polishing this feature in the coming months including adding telemetry support for TCP services when Istio mTLS is being used. 
```

Comment:
Make it clear that it is *experimental* in 1.3. 
`*Mixer-less http telemetry (experimental)*`